db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20210407170804) do
+ActiveRecord::Schema.define(version: 20210326094519) do
```

Comment:
Just checking the rebase with `schema.rb`... I think this needs to be the higher of the two. It should be `20210407170804`.

Reply:
https://github.com/openfoodfoundation/openfoodnetwork/pull/7430

Reply:
Oops ... sorry and thanks for that!